src/stories/epics/createForm.stories.tsx

```diff
@@ -17,10 +17,15 @@ const Template: Story<ComponentProps<typeof CreateEpicModalComponent>> = (
   args,
 ) => <CreateEpicModalComponent {...args} />;
 
-export const Component = Template.bind({});
-Component.args = {
+export const CreateEpicModal = Template.bind({});
+CreateEpicModal.args = {
   user: sampleUser1,
   project: sampleProject1,
   isOpen: true,
   closeCreateModal: action('closeCreateModal'),
 };
+CreateEpicModal.argTypes = {
+  user: { control: { disable: true } },
+  project: { control: { disable: true } },
+};
+CreateEpicModal.storyName = 'Example';
```

Comment:
```suggestion
export const CreateForm = Template.bind({});
CreateForm.args = {
  user: sampleUser1,
  project: sampleProject1,
  isOpen: true,
  closeCreateModal: action('closeCreateModal'),
};
CreateForm.argTypes = {
  user: { control: { disable: true } },
  project: { control: { disable: true } },
};
CreateForm.storyName = 'Example';
```

Reply:
I realize this might not be all needed to change its name, but suggesting the name change to `CreateForm`

Reply:
I agree this make the naming here more consistent.